series-tmdb.en.yaml

```diff
@@ -0,0 +1,2683 @@
+      - season: 1
+        anilist-id: 117533
+
+  - title: "Chaika - The Coffin Princess"
+    # https://www.themoviedb.org/tv/60825
+    # https://www.thetvdb.com/dereferrer/series/278127
+    # https://www.imdb.com/title/tt3517016/
+    synonyms:
+      - "Chaika the Coffin Princess"
+      - "Hitsugi no Chaika"
+      - "棺姫のチャイカ AVENGING BATTLE"
+    seasons:
+      - season: 1
+        anilist-id: 20462
+      - season: 2
+        anilist-id: 20719
+
+  - title: "Chainsaw Man"
+    # https://www.themoviedb.org/tv/114410
+    # https://www.thetvdb.com/dereferrer/series/397934
+    # https://www.imdb.com/title/tt13616990/
+    synonyms:
+      - "CSM"
+      - "Chainsaw.Man"
+      - "Chainsawman"
+    seasons:
+      - season: 1
+        anilist-id: 127230
+
+  - title: "Clannad"
+    # https://www.themoviedb.org/tv/24835
+    # https://www.thetvdb.com/dereferrer/series/80644
+    # https://www.imdb.com/title/tt1118804/
+    synonyms:
+      - "Kuranado"
+      - "クラナド"
+    seasons:
+      - season: 1
+        anilist-id: 2167
+      - season: 2
+        anilist-id: 4181
+
+  - title: "Code Geass: Lelouch of the Rebellion"
+    # https://www.themoviedb.org/tv/31724
+    # https://www.thetvdb.com/dereferrer/series/79525
+    # https://www.imdb.com/title/tt0994314/
+    synonyms:
+      - "Code Geass"
+      - "Code Geass Hangyaku no Lelouch"
+      - "Code Geass: Hangyaku no Lelouch"
+    seasons:
+      - season: 1
+        anilist-id: 1575
+      - season: 2
+        anilist-id: 2904
+
+  - title: "D.Gray-man"
+    # https://www.themoviedb.org/tv/34141
+    # https://www.thetvdb.com/dereferrer/series/79635
+    # https://www.imdb.com/title/tt0899258/
+    synonyms:
+      - "D.Gray-man"
+      - "ディー・グレイマン"
+    seasons:
+      - season: 1
+        anilist-id: 1482
+      - season: 2
+        anilist-id: 1482
+        start: 52
+
+  - title: "D.Gray-man Hallow"
+    # https://www.themoviedb.org/tv/67145
+    # https://www.thetvdb.com/dereferrer/series/312878
+    # https://www.imdb.com/title/tt5954268/
+    synonyms:
+      - "D.Gray-man HALLOW"
+      - "ディーグレイマンハロー"
+      - "ディー・グレイマン ハロー"
+    seasons:
+      - season: 1
+        anilist-id: 21558
+
+  - title: "Danganronpa: The Animation"
+    # https://www.themoviedb.org/tv/49471
+    # https://www.thetvdb.com/dereferrer/series/270058
+    # https://www.imdb.com/title/tt2902582/
+    synonyms:
+      - "Danganronpa"
+      - "Danganronpa The Animation"
+      - "ダンガンロンパ"
+    seasons:
+      - season: 1
+        anilist-id: 16592
+      - season: 2
+        anilist-id: 21509
+
+  - title: "Danganronpa 3: The End of Hope's Peak High School"
+    # https://www.themoviedb.org/tv/67674
+    # https://www.imdb.com/title/tt6027642/
+    synonyms:
+      - "Danganronpa 3 The End Of Kibougamine Gakuen Zetsubou Hen"
+      - "Danganronpa 3 Zetsubou Hen"
+    seasons:
+      - season: 1
+        anilist-id: 21825
+
+  # PlexAniSync does not support mapping of Specials (Gaiden, 4 episodes)
```

Comment:
Comment needs to be under title so sort.py doesn't associate it with Danganronpa 3.